### PR TITLE
Adding additional instructions for windows users when installing wrangler

### PR DESCRIPTION
**Pull request by silvererudite**
I ran into some unnecessary trouble while running wrangler for the first time because it wasn't added to my computer's PATH. So I thought maybe it would be a good idea to add this information on the docs.

### CODE REVIEW

**Comment by signalnerve** [comment]
i think this is a super useful change, i'm just not sure where the best place is for it. cc @cloudflare/workers-devexp – my question is if this should be in the docs or better suited for the wrangler repo. thanks for submitting a pr!!

**Comment by silvererudite** [comment]
hi @signalnerve would be happy to modify any changes requested. Let me know if I need to move this PR to where  it's better suited.

File: src/content/tooling/wrangler/install.md [hunk 1]
```diff
@@ -65,3 +65,6 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
 1. Download the binary tarball for your platform from our [releases page](https://github.com/cloudflare/wrangler/releases). You don't need to download wranglerjs, wrangler will install that for you.
 
 2. Unpack the tarball and place the binary `wrangler` somewhere on your `PATH`, preferably `/usr/local/bin` for linux/macOS or `Program Files` for windows.
+
+### Note for Windows users
+After installing wrangler, you might need to add it to you Environment path if you don't have already.Otherwise you might face some unexpected problems when you run wrangller commands in you command prompt. For more info on how to do so you can follow this  [tutorial here](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/)
```
Comment (victoriabernard92) [inline src/content/tooling/wrangler/install.md:70]:
```suggestion
Wrangler needs to be in your path to run from the command line. Window's users see [How to Add to Windows PATH Environment Variable](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/)
```

**Review by exvuma — CHANGES_REQUESTED** [review]
Thanks for this! I think we can make this more concise without loosing information. Make sure to fix merge conflicts and run through a grammar checker for the extra spaces then happy to review again.

**Comment by rita3ko** [comment]
Closing as we've migrated to a new repo, I opened an issue (https://github.com/cloudflare/cloudflare-docs/issues/32) since this is a great tip to document.

**Pull request closed without merging**